feed_flat_test.go

```diff
@@ -8,16 +8,18 @@ import (
 	getstream "github.com/GetStream/stream-go"
 )
 
-func getFlatFeed(client *getstream.Client) *getstream.FlatFeed {
-	f, _ := client.FlatFeed("flat", RandString(8))
-	return f
+func getFlatFeed(client *getstream.Client) (*getstream.FlatFeed, error) {
+	return client.FlatFeed("flat", RandString(8))
```

Comment:
I would pass `t` and do `t.Fatal` here if there's an error. Causes a lot less error checking clutter in the tests.